guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/repository/wizard/CreateRepositoryWizardTest.java

```diff
@@ -569,6 +573,230 @@ public void testManagedRepositoryMultiCompletedClashingGAVForcedTest() {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
+    public void testManagedRepositoryMultiCompletedClashingGAVBeforeRepositoryInitializationTest() {
```

Comment:
@triceo Glad you like it - when I start to need to break method names over 2 lines I think I'll need a new strategy!